portal/app/algorithms/battery_load_control.py

```diff
@@ -0,0 +1,52 @@
+import json, datetime
+import numpy as np
+from app.farm_updater import sonnen_api
+
+
+def batt_dispatch():
+    from app.models import FarmDevice, FarmData, DeviceType
+
+    egauge_device = FarmDevice.objects.get(device_uid='46613')
```

Comment:
I don't remember on what we settled but why can't this id be in the settings file?